rpi-update

```diff
@@ -115,10 +115,39 @@ function update_sdk {
 }
 
 function show_notice {
-	local NOTICE=`curl -Lfs https://raw.githubusercontent.com/hexxeh/rpi-firmware/${FW_REV}/NOTICE.md`
+	local FULL_NOTICE=`curl -Lfs https://raw.githubusercontent.com/hexxeh/rpi-firmware/${FW_REV}/NOTICE.md`
+	local NOTICE=$(echo "$FULL_NOTICE" | tail -n+2)
 	if [ -z "$NOTICE" ]; then
 		return
 	fi
+	local NOTICE_HASH_HEAD=$(echo "$FULL_NOTICE" | head -1)
+	if [ $(echo "${NOTICE_HASH_HEAD}" | awk -F: '{print $1}') == "HASH" ]; then
+		local NOTICE_HASH_EXISTS=true
+		local NOTICE_HASH=$(echo "${NOTICE_HASH_HEAD}" | awk '{print $2}')
+		local NOTICE_HASH_DATE=$(get_hash_date ${NOTICE_HASH})
+	else
+		local NOTICE_HASH_EXISTS=false
+	fi
+	if [ -f "$FW_REVFILE" ]; then
+		local LOCAL_HASH=$(cat "$FW_REVFILE")
+	else
+		local LOCAL_HASH=0
+		local LOCAL_HASH_DATE=0
+	fi
+	if ${NOTICE_HASH_EXISTS}; then
+		local NEW_HASH=${FW_REV}
+		local NEW_HASH_DATE=$(get_hash_date ${NEW_HASH})
+		if [ -f "$FW_REVFILE" ]; then
+			local LOCAL_HASH_DATE=$(get_hash_date ${LOCAL_HASH})
+		fi
+		local LOCAL_lt_NOTICE=$(compare_hashes "${LOCAL_HASH_DATE}" lt "${NOTICE_HASH_DATE}")
+		local NEW_ge_NOTICE=$(compare_hashes "${NEW_HASH_DATE}" ge "${NNOTICE_HASH_DATE}")
```

Comment:
Just spotted that this line has `${NNOTICE_HASH_DATE}` (note the two Ns) which I guess is a typo.